src/fairseq2/nn/padding.py

```diff
@@ -187,7 +187,10 @@ def get_seqs_and_padding_mask(
 
 
 def pad_seqs(
-    seqs: Sequence[Tensor], pad_value: int = 0, pad_to_multiple: int = 1
+    seqs: Sequence[Tensor],
+    pad_value: int = 0,
+    pad_to_multiple: int = 1,
+    device: Device | None = None,
```

Comment:
@cbalioglu : are you that we add this extra argument here ?

Reply:
looks good to me.